examples/create_rosa_hcp_cluster/hcp_cluster_with_managed_oidc_config/README.md

```diff
@@ -0,0 +1,127 @@
+# Creating a ROSA HCP cluster with a managed OIDC configuration ID
```

Comment:
I prefer you will move that example dir into the new [module](https://github.com/terraform-redhat/terraform-rhcs-rosa)